keystore/src/iosMain/kotlin/com/b1nd/dodam/keystore/KeyStoreManager.ios.kt

```diff
@@ -0,0 +1,11 @@
+package com.b1nd.dodam.keystore
+
+actual class KeyStoreManager actual constructor() {
+    actual fun encrypt(plainText: String): String {
+        TODO("Not yet implemented")
```

Comment:
please implementation this fun

Reply:
The iOS version of Keychain will be implemented in another branch.